crypto/pem/pvkfmt.c

```diff
@@ -795,16 +795,19 @@ static int derive_pvk_key(unsigned char *key,
                           const unsigned char *pass, int passlen)
 {
     EVP_MD_CTX *mctx = EVP_MD_CTX_new();
+    EVP_MD *md = EVP_MD_fetch(NULL, SN_sha1, NULL);
```

Comment:
Is this method legacy only? (i.e no libctx)

Reply:
I've no idea :(

Reply:
It's guarded by a `#ifndef OPENSSL_NO_RC4`.  I'm not seeing a compelling need to introduce an _ex flavour.
Moreover, none of its callers have access to a libctx.

It could be pulled out as a standalone KDF I guess.  I'm not seeing much value to doing so.